telemetry/roles/prometheus_gaudi/tasks/deploy_gaudi_metric_exporter.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+
+# Assuming Prometheus was deployed and name space monitoring was created
```

Comment:
create roles/prometheus_gaudi/tasks/main.yml and call this deploy_gaudi_metric_exporter.yml from main. yml

Reply:
done